Cannot successfully run `make test-e2e-local-simple`

**How to categorize this issue?**
<!--
Please select area, kind, and priority for this issue. This helps the community categorizing it.
Replace below TODOs or exchange the existing identifiers with those that fit best in your opinion.
If multiple identifiers make sense you can also state the commands multiple times, e.g.
  /area control-plane
  /area auto-scaling
  ...

"/area" identifiers:     audit-logging|auto-scaling|backup|certification|control-plane-migration|control-plane|cost|delivery|dev-productivity|disaster-recovery|documentation|high-availability|logging|metering|monitoring|networking|open-source|ops-productivity|os|performance|quality|robustness|scalability|security|storage|testing|usability|user-management
"/kind" identifiers:     api-change|bug|cleanup|discussion|enhancement|epic|impediment|poc|post-mortem|question|regression|task|technical-debt|test
-->
/area dev-productivity
/kind bug
/kind regression

**What happened**:
Running `make test-e2e-local-simple` results in 
```
% make test-e2e-local-simple KUBECONFIG="$PWD/example/gardener-local/kind/local/kubeconfig"
./hack/test-e2e-local.sh --procs=5 --label-filter "Shoot && simple" ./test/e2e/gardener/...
> E2E Tests
Hostnames for Shoot e2e-update-node.local is missing in /etc/hosts. To access shoot clusters and run e2e tests, you have to extend your /etc/hosts file.
Please refer to https://github.com/gardener/gardener/blob/master/docs/deployment/getting_started_locally.md#accessing-the-shoot-cluster

make: *** [test-e2e-local-simple] Error 1
```

Then I go to https://github.com/gardener/gardener/blob/master/docs/deployment/getting_started_locally.md#accessing-the-shoot-cluster, run the command to update my `/etc/hosts` file. Then I re-run the `make test-e2e-local-simple` command but it keeps failing the same way.

The regression is introduced with https://github.com/gardener/gardener/pull/7122.

**What you expected to happen**:
To be able to run `make test-e2e-local-simple`.

**How to reproduce it (as minimally and precisely as possible)**:
See above.
**Anything else we need to know?**:

**Environment**:

- Gardener version: 6d18b043a3852034e571ccc9f5e265db1923fd45
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration:
- Others:

--- a/docs/deployment/getting_started_locally.md
+++ b/docs/deployment/getting_started_locally.md
@@ -121,10 +121,10 @@ cat <<EOF | sudo tee -a /etc/hosts
 127.0.0.1 api.e2e-rotate.local.internal.local.gardener.cloud
 127.0.0.1 api.e2e-default.local.external.local.gardener.cloud
 127.0.0.1 api.e2e-default.local.internal.local.gardener.cloud
-127.0.0.1 api.e2e-udpate-node.local.external.local.gardener.cloud
-127.0.0.1 api.e2e-udpate-node.local.internal.local.gardener.cloud
-127.0.0.1 api.e2e-udpate-zone.local.external.local.gardener.cloud
-127.0.0.1 api.e2e-udpate-zone.local.internal.local.gardener.cloud
+127.0.0.1 api.e2e-update-node.local.external.local.gardener.cloud
+127.0.0.1 api.e2e-update-node.local.internal.local.gardener.cloud
+127.0.0.1 api.e2e-update-zone.local.external.local.gardener.cloud
+127.0.0.1 api.e2e-update-zone.local.internal.local.gardener.cloud
 127.0.0.1 api.e2e-upgrade.local.external.local.gardener.cloud
 127.0.0.1 api.e2e-upgrade.local.internal.local.gardener.cloud
 EOF

